src/main/java/frc/robot/commands/ArcadeDrive.java

```diff
@@ -8,27 +8,47 @@
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.Constants;
 import frc.robot.subsystems.Drivetrain;
-
 public class ArcadeDrive extends CommandBase {
   /** Creates a new ArcadeDrive. */
-  private Drivetrain drivetrain;
-  private Joystick js1;
-  public ArcadeDrive(Drivetrain drive) {
+  private Drivetrain m_drivetrain;
+  private Joystick m_joystick;
+  public ArcadeDrive(Drivetrain drive, Joystick joy) {
     // Use addRequirements() here to declare subsystem dependencies.
     addRequirements(drive);
-    drivetrain = drive;
+    m_drivetrain = drive;
+    m_joystick = joy;
   }
 
   // Called when the command is initially scheduled.
   @Override
-  public void initialize() {
-    js1 = new Joystick(Constants.Ports.JOYSTICK_1);
-  }
-
+  public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    drivetrain.betterArcadeDrive(js1.getRawAxis(Constants.Ports.JOYSTICK_1), js1.getRawAxis(Constants.Ports.JOYSTICK_2));
+
+    double leftRawInput = m_joystick.getRawAxis(Constants.Controller.JOYSTICK_1);
+    double rightRawInput = m_joystick.getRawAxis(Constants.Controller.JOYSTICK_2);
+    
+    double leftFinalInput = 0;
+    double rightFinalInput = 0;
+
+    if (leftRawInput > Constants.Controller.DEAD_ZONE_SENSITIVITY && leftRawInput < 1) {
+        if (leftRawInput < 1-Constants.MovementConstants.INPUT_SMOOTH_SLOPE) {
+            leftFinalInput += Constants.MovementConstants.INPUT_SMOOTH_SLOPE;
+        }
+        else {
+            leftFinalInput = 1;
+        }
+    }
+    if (rightRawInput < Constants.Controller.DEAD_ZONE_SENSITIVITY*-1 && rightRawInput > -1) {
+        if (rightRawInput < 1-Constants.MovementConstants.INPUT_SMOOTH_SLOPE) {
```

Comment:
if (rightRawInput < 1-Constants.MovementConstants.INPUT_SMOOTH_SLOPE) {
Should this be "if (rightRawInput > Constants.MovementConstants.INPUT_SMOOTH_SLOPE - 1) {"?